contributing/community/mentoring.rst

```diff
@@ -0,0 +1,12 @@
+Mentoring
+=========
+
+Reading the :doc:`contributing </contributing/index>` is already a great way
+to get started on becoming a Symfony contributor. However, sometimes
+it might still seem overwhelming and for this purpose we created a
+dedicated slack channel called `contribs`_ to connect new contributors
+to long time contributors that can help get started and give some one
```

Comment:
someone (one word), and the whole sentence seems a bit strange.

started and give someone advice on the entire process.

Reply:
> long-time contributors. This is a great way to get one-on-one advice on the entire process.
> These long-time contributors *do* really want to help new contributors - so feel free to ask anything!